packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -535,46 +597,99 @@ contract L2StandardBridge_Bridge_Test is Bridge_Initializer {
-        vm.expectRevert("StandardBridge: cannot bridge ETH with custom gas token");
 
-        l2StandardBridge.bridgeETHTo(bob, 60000, hex"dead");
+        l2StandardBridge.bridgeETH{ value: _value }(_minGasLimit, _extraData);
     }
 
-    /// @dev Tests that bridging ETH reverts if a custom gas token is used.
-    function test_bridgeETH_isCustomGasToken_reverts() external {
+    /// @dev Tests that bridging reverts with custom gas token.
+    function test_bridgeETH_customGasToken_reverts() external {
         vm.prank(alice, alice);
         vm.mockCall(address(l1Block), abi.encodeWithSignature("gasPayingToken()"), abi.encode(address(1), uint8(2)));
         vm.expectRevert("StandardBridge: cannot bridge ETH with custom gas token");
 
         l2StandardBridge.bridgeETH(50000, hex"dead");
     }
 
-    /// @dev Tests that the receive function reverst if a custom gas token is used.
-    function test_receive_isCustomGasToken_reverts() external {
+    /// @dev Tests that bridging ETH to a different address succeeds.
+    function testFuzz_bridgeETHTo_succeeds(uint256 _value, uint32 _minGasLimit, bytes calldata _extraData) external {
+        uint256 nonce = l2CrossDomainMessenger.messageNonce();
+
+        vm.expectCall(
+            address(l2StandardBridge),
+            _value,
+            abi.encodeWithSelector(l1StandardBridge.bridgeETHTo.selector, bob, _minGasLimit, _extraData)
+        );
+
+        bytes memory message =
+            abi.encodeWithSelector(StandardBridge.finalizeBridgeETH.selector, alice, bob, _value, _extraData);
+
+        // the L2 bridge should call
+        // L2CrossDomainMessenger.sendMessage
+        vm.expectCall(
+            address(l2CrossDomainMessenger),
+            abi.encodeWithSelector(
+                CrossDomainMessenger.sendMessage.selector, address(l1StandardBridge), message, _minGasLimit
+            )
+        );
+
+        vm.expectEmit(address(l2StandardBridge));
+        emit ETHBridgeInitiated(alice, bob, _value, _extraData);
+
+        // SentMessage event emitted by the CrossDomainMessenger
+        vm.expectEmit(address(l2CrossDomainMessenger));
+        emit SentMessage(address(l1StandardBridge), address(l2StandardBridge), message, nonce, _minGasLimit);
+
+        // SentMessageExtension1 event emitted by the CrossDomainMessenger
+        vm.expectEmit(address(l2CrossDomainMessenger));
+        emit SentMessageExtension1(address(l2StandardBridge), _value);
+
+        // deposit eth to bob
+        vm.deal(alice, _value);
         vm.prank(alice, alice);
-        vm.mockCall(address(l1Block), abi.encodeWithSignature("gasPayingToken()"), abi.encode(address(1), uint8(2)));
 
+        l2StandardBridge.bridgeETHTo{ value: _value }(bob, _minGasLimit, _extraData);
+    }
+
+    /// @dev Tests that bridging reverts with custom gas token.
+    function test_bridgeETHTo_customGasToken_reverts() external {
```

Comment:
This could be good to fuzz to get value and non value

Reply:
I thought we'd just revert for all values with custom gas token? if so, I don't think it'd make sense to fuzz?

Reply:
we do revert for all values but right now the test only covers one of the two cases